digitkraft-backend/src/main/resources/db/migration/h2/V1.0__create_schema.sql

```diff
@@ -0,0 +1,162 @@
+CREATE SEQUENCE hibernate_sequence START WITH 1 INCREMENT BY 1;
+
```

Comment:
I added in this migrations changes for test H2 table. When you will be modyfying existing entities remember to also update this script. You can use Spring JPA Buddy, which I highly recommend.